docs/source/conf.py

```diff
@@ -28,6 +28,8 @@
 sys.path.insert(0, os.path.abspath(PATH_ROOT))
 sys.path.append(os.path.join(PATH_RAW_NB, '.actions'))
 
+# if you run into an import error here, try to execute this command:
+# git submodule update --init --recursive
 from helpers import HelperCLI  # noqa: E401 E402
```

Comment:
@Borda Should we add a try-except here?

Reply:
btw I added this comment <3 
if running into this issue, the stack trace will lead you to this comment. 
a try except would also work for a more user friendly message :)